### ISSUE TITLE
MO2 what to do with Overwrite files? (Dec 2021)

### ISSUE THREAD

**Issue by pgorbas** [ModOrganizer2/modorganizer#1625]
Hello, I'm new to MO2 and was happily installing mods into Skyrim SE (Avoiding AE for now until more mods are updated) then I noticed I am getting some files in the overwrite section.

OLD out of date documentation maintained by MO2, about 6 or 7 years out of date, says to right click on the files and select create mod - that's great except there is no longer a "create mod" command.  

There is now no guidance for this issue.  There is a right-click command "Create Backup" which, surprisingly not only creates _another _file__int he overwrite directory, it leaves the original file in the overwrite directory as well.  So no help there, in fact it just makes the problem worse.

**Some Guidance would be appreciated!**

**Comment by Qudix** [comment]
Still there
![image](https://user-images.githubusercontent.com/17361645/145936230-167dfcaf-b203-41be-b720-c0d546c0d464.png)

**Comment by pgorbas** [comment]
You are NOT right clicking on the files as the instructions in the documentation says to do
![M02](https://user-images.githubusercontent.com/11726837/145946429-786612bd-0170-47b1-b28f-a272e0f79dd6.jpg)
;

**Comment by pgorbas** [comment]
Thank you for your response.  It does not appear that you are right clicking on the file names as the documentation tells me to do:


-Regards, Paul Gorbas 

    On Monday, December 13, 2021, 10:03:00 PM MST, Qudix ***@***.***> wrote:  
 
 


Still there


—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Al12rs** [comment]
What documentation are you referring to? In app documentation or some site?

If you want to know how to handle overwrite files first try to understand what overwrite means and why it is better to keep it clean, then it's just a matter of moving the files from overwrite to a new location.

Here is our wiki on overwrite: https://github.com/ModOrganizer2/modorganizer/wiki/Troubleshooting#there-are-files-in-your-overwrite-mod-directory

**Comment by LostDragonist** [comment]
> You are NOT right clicking on the files as the instructions in the documentation says to do ![M02](https://user-images.githubusercontent.com/11726837/145946429-786612bd-0170-47b1-b28f-a272e0f79dd6.jpg) ;

You need to right-click on the overwrite mod, not some other mod...
